packages/core/src/config/parse.ts

```diff
@@ -962,8 +1022,9 @@ const parseContractVariables = (
 export const parseAndValidateConstructorArgs = (
   userConstructorArgs: UserConfigVariables,
   referenceName: string,
-  abi: Array<Fragment>
-): ParsedConfigVariables => {
+  abi: Array<Fragment>,
+  cre: ChugSplashRuntimeEnvironment
+): { args: ParsedConfigVariables } => {
```

Comment:
> > I think this can be simplified to `return parsedConfigVariables`
> 
> Similarly, I think this can be simplified from `{ variables: ParsedConfigVariables }` to just `ParsedConfigVariables`

Same thing here